SparkPost/2web2ui
# AC-1388 -  (UI)Testing asking for domain during set up (like we used to)

AC-1388 -  (UI)Testing asking for domain during set up (like we used to)

### What Changed
- Added back following 
      - SendingDomain Page
      - Cypress tests and unit tests related to SendingDomainPage
      - Added in a variable (`window.skipSendingDomain `) to control whether user sees the Sending Domain step or not.
    

### How To Test
- Set window.skipSendingDomain to false in console, then this should be the signup flow
     signup > planpicker > DOMAIN > Dashboard

- Set window.skipSendingDomain to true in console, then this should be the signup flow
     signup > planpicker > Dashboard

### To Do
